lib/event-service.js

```diff
@@ -0,0 +1,59 @@
+var EVENTSERVICE_SERVICE_UUID          = 'E95D93AF251D470AA062FA1922DFA9A8'.toLowerCase();
+var MICROBITEVENT_CHARACTERISTIC_UUID = 'E95D9775251D470AA062FA1922DFA9A8'.toLowerCase();
+
+var CLIENTREQUIREMENTS_CHARACTERISTIC_UUID = 'E95D23C4251D470AA062FA1922DFA9A8'.toLowerCase();
+var CLIENTEVENT_CHARACTERISTIC_UUID = 'E95D5404251D470AA062FA1922DFA9A8'.toLowerCase();
+
+var microbit_event_subscribed = 0;

```

Comment:
No worries. As you've doubtless deduced by now, as a node.js developer I am a complete amateur :-) Trying to make a contribution though. OK, come the weekend hopefully I'll get to this and take another pass.